### ISSUE TITLE
Delete this Repo

### ISSUE THREAD

**Issue by simsong** [M2Crypto/M2Crypto#38]
If this repo is dead and not maintained, it should be deleted.

**Comment by mcepl** [comment]
Yes, it should ... but I need @martinpaljak 's cooperation for it, and I have been trying in vane for a year to get it. I have asked support@github.com to do something about it now.

**Comment by simsong** [comment]
You may wish to leave this ticket open!

**Comment by idgserpro** [comment]
I know it's not going to be easy, but we think you shouldn't lose the fixes that were commited to this repository if martinpaljak is going to be the canonical one.

**Comment by simsong** [comment]
You won't lose the fixes. Just merge them in to your canonical repository.

**Comment by idgserpro** [comment]
We're already using martinpaljak master, we just [suggested to merge https://github.com/M2Crypto/M2Crypto/ and https://github.com/martinpaljak/M2Crypto](https://github.com/M2Crypto/M2Crypto/issues/36). Although we can have a fork I don't think it's the best approach and would lead to even more confusion.

In https://pypi.python.org/pypi/M2Crypto/0.22.3 it says

```
Maintainer: Martin Paljak
Home Page: https://github.com/M2Crypto/M2Crypto
```

so we would like to solve this confusion in M2Crypto or martinpaljak.

**Comment by mcepl** [comment]
I have no control over the Martin's repo or over PyPI.
